packages/blockchain-wallet-v4-frontend/src/modals/Mobile/MobileNumberAdd/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators, compose } from 'redux'
+
+import { actions } from 'data'
+import modalEnhancer from 'providers/ModalEnhancer'
+
+import { getData } from './selectors'
+import MobileNumberAdd from './template.js'
+
+class MobileNumberAddContainer extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.onSubmit = this.onSubmit.bind(this)
+  }
+
+  onSubmit() {
+    const { smsNumberNew } = this.props
```

Comment:
do we validate SMS number somehow?

Reply:
I have no idea. It doesn't seem so. These changes were made by the linter